lib/workload/stateless/stacks/workflow-manager/workflow_manager/models/state.py

```diff
@@ -89,27 +90,26 @@ class State(OrcaBusBaseModel):
     class Meta:
         unique_together = ["workflow_run", "status", "timestamp"]
 
-    id = models.BigAutoField(primary_key=True)
+    orcabus_id_prefix = 'stt.'
 
     # --- mandatory fields
-    workflow_run = models.ForeignKey(WorkflowRun, related_name='states', on_delete=models.CASCADE)
     status = models.CharField(max_length=255)  # TODO: How and where to enforce conventions?
     timestamp = models.DateTimeField()
-
     comment = models.CharField(max_length=255, null=True, blank=True)
 
+    workflow_run = models.ForeignKey(WorkflowRun, related_name='states', on_delete=models.CASCADE)
     # Link to workflow run payload data
     payload = models.ForeignKey(Payload, null=True, blank=True, on_delete=models.SET_NULL)
 
     objects = StateManager()
 
     def __str__(self):
-        return f"ID: {self.id}, status: {self.status}"
+        return f"ID: {self.orcabus_id}, status: {self.status}"
 
     def to_dict(self):
         return {
-            "id": self.id,
-            "workflow_run_id": self.workflow_run.id,
+            "orcabusId": self.orcabus_id,
```

Comment:
Should this be `snake_case`?

Reply:
Fair enough. Not even sure those methods are needed at all.
I'll just remove them.